submissions/YuliiaHordiichuk/html-movie-seat-booking/index.html

```diff
@@ -0,0 +1,355 @@
+    <title>Cinema App</title>
+</head>
+
+<body>
+    <section class="booking-page container" id="booking-page">
+        <header class="booking-page__header"> 
+            <h1 class="booking-page__title">DIXON</h1>
+            <nav class="booking-page__nav nav">
+                <ul class ="nav__list">
+                    <li class="nav__item">
+                        <span class="nav__accent">01</span> 
+                        <span>choose a movie </span>
+                    </li>
+                    <li class="nav__item">
+                        <span class="nav__accent">02</span> 
+                        <span>choose seats</span>    
+                    </li>
+                    <li class="nav__item">
+                        <span class="nav__accent">03</span>
+                        <span>purchase</span>  
+                    </li>
+                    <li class="nav__item">
+                        <span class="nav__accent">04</span>
+                        <span>complete</span>  
+                    </li>
+                </ul>
+            </nav>
+        </header>
+
+        <aside class="booking-page__aside poster">
+            <img src="./img/poster.jpg" alt="Movie poster" class="poster__img">
+
+            <ul class="poster__list">
+                <li class="poster__item">
+                    <h3 class="poster__heading">Director</h3>
+                    <p class="poster__text">Todd Phillips</p>
+                </li>
+                <li class="poster__item">
+                    <h3 class="poster__heading">Starring</h3>
+                    <p class="poster__text"> Joaquin Phoenix, Robert De Niro, Zazie Beetz</p>
+                </li>
+                <li class="poster__item">
+                    <h3 class="poster__heading">Genre</h3>
+                    <p class="poster__text">Crime, Drama, Thriller</p>
+                </li>
+            </ul>
+        </aside>
+
+        <div class="booking-page__wrapper">
+            <section class="booking-page__main main">
+
+                <header class="main__header">
+                    <h2 class="main__title">JOKER</h2>
+                    <div class="main__desc">
+                        <i class="far fa-clock main__time-icon"></i>
+                        <span class="main__duration">122 minutes</span>
+                        <span class="main__hall">6-RED</span>
+                    </div>
+                </header>
+
+                <div class="main__info info">
+                    <div class="info__details info__details_day">
+                        <p class="info__title">Monday, 1 February</p>
+                        <ul class="info__list">
+                            <li class="info__item info__item_button">
+                                <button class="info__button">
+                                    <i class="fas fa-long-arrow-alt-left"></i>
+                                </button>
+                            </li>
```

Comment:
The questionable decision put buttons into the list.

Reply:
Changed :)